content/en/account_management/teams.md

```diff
@@ -0,0 +1,106 @@
+
+1. On the [Team directory page][1], click **New Team** at the upper right.
+1. Choose a **Team Name**.
+1. The **Handle** populates based on your team name.
+1. Optionally, choose an **Avatar Color**.
+1. Provide an optional **Summary**.
+1. Use the drop-down to select team members.
+1. Click **Create**.
+
+### Modify team
+
+1. On the [Team directory page][1], click the team you would like to modify. A side panel appears with the team details.
+1. Mouse over the item you wish to modify. A pencil icon appears.
+1. Click the pencil icon. A pop-up window appears.
+1. Make your changes, then click the appropriate button to save your changes.
+
+## Team handle
+
+A team handle links teams to Datadog resources. Team handles appear in search bars and facets in the format `team:<team-name>`. 
+
+To find a team handle:
+1. Click the team's name in the Team directory page. A side panel appears with team details.
+1. Look for the **handle** field at the top of the panel. 
+
+To associate a resource with a defined team, a Team must exist in Datadog with a matching team handle. When you click on a resource associated with a defined team, a small window appears with the team handle and additional information. Defined teams provide additional functionality such as the Team filter below. 
+
+Team handles that aren’t associated with a defined team in Datadog have a tag-like experience. Convert any undefined team handles to defined teams to take advantage of Teams features. 
+
+### Associate resources with team handles
+
+Datadog supports associating the following resources with team handles:
+
+- Dashboards
+- Service Catalog
+- Incidents
+- Monitors
+
+## Filter
+
+The team filter tailors user experiences to content associated with their team.
+
+The team filter appears in two places in each list view: 
+- A list of search facets at the top left
+- A search term in the search bar
+
+Enabling a team filter refines the list of resources based on a user's team memberships or the services owned by their teams. The team filter provides a global and persistent context that follows a user’s journey across products.
+
+The table below describes the products in which you can use the team filter:
+
+| Product List Page       | Filter basis                                                                     |
+|-------------------------|----------------------------------------------------------------------------------|
+| [Dashboards][2]         | Team handle                                                                      |
+| [Service Catalog][3]    | Team handle                                                                      |
+| [Incidents][4]          | Team handle                                                                      |
+| [Monitors][5]           | Team handle                                                                      |
+| [APM Error Tracking][6] | Service owned by teams (determined by ownership inside the [Service Catalog][3]) |
```

Comment:
[Logs Error Tracking](https://app.datadoghq.com/logs/error-tracking) also has the Team Filter and filters by the same mechanism as APM Error Tracking